Project2015To2017/Reading/ProjectPropertiesReader.cs

```diff
@@ -79,20 +92,278 @@ public static void PopulateProperties(Project project, XDocument projectXml)
+						case "SignAssembly" when ValidateDefaultValue("false"):
+						case "DelaySign" when ValidateDefaultValue("false"):
+						case "GeneratePackageOnBuild" when ValidateDefaultValue("false"):
+						case "PackageRequireLicenseAcceptance" when ValidateDefaultValue("false"):
+						case "DebugSymbols" when ValidateDefaultValue("false"):
+						case "CheckForOverflowUnderflow" when ValidateDefaultValue("false"):
+						case "AppendTargetFrameworkToOutputPath" when ValidateDefaultValue("true"):
+						case "AppDesignerFolder" when ValidateDefaultValue("properties"):
+						case "DefaultProjectTypeGuid" when ValidateDefaultValue("{fae04ec0-301f-11d3-bf4b-00c04f79efbc}"):
+						case "DefaultLanguageSourceExtension" when ValidateDefaultValue(".cs"):
+						case "Language" when ValidateDefaultValue("C#"):
+						case "TargetRuntime" when ValidateDefaultValue("managed"):
+						case "Utf8Output" when ValidateDefaultValue("true"):
+						case "PlatformName" when ValidateDefaultValue("$(platform)")
+						                         || (parentConditionHasPlatform &&
+						                             ValidateDefaultValue(parentConditionPlatformLower)):
+						// Conditional platform default values
+						case "PlatformTarget" when parentConditionHasPlatform && child.Value == parentConditionPlatform
+						                                                      && !hasGlobalOverride:
+						// Conditional configuration (Debug/Release) default values
+						case "DefineConstants" when isDebugOnly && ValidateDefaultConstants(valueLower, "debug", "trace") &&
+						                            !hasGlobalOverride:
+						case "DefineConstants" when isReleaseOnly && ValidateDefaultConstants(valueLower, "trace") &&
+						                            !hasGlobalOverride:
+						case "Optimize" when isDebugOnly && valueLower == "false" && !hasGlobalOverride:
+						case "Optimize" when isReleaseOnly && valueLower == "true" && !hasGlobalOverride:
+						case "DebugSymbols" when isDebugOnly && valueLower == "true" && !hasGlobalOverride:
+						// Default project values for Platform and Configuration
+						case "Platform" when !hasParentCondition && ValidateEmptyConditionValue(childCondition) &&
+						                     valueLower == "anycpu":
+						case "Configuration" when !hasParentCondition && ValidateEmptyConditionValue(childCondition) &&
+						                          valueLower == "debug":
+						// Extra ProjectName duplicates
+						case "RootNamespace" when !hasParentCondition && ValidateEmptyConditionValue(childCondition) &&
+						                          child.Value == projectName:
+						case "AssemblyName" when !hasParentCondition && ValidateEmptyConditionValue(childCondition) &&
+						                         child.Value == projectName:
+						{
+							removeQueue.Add(child);
+							break;
+						}
+
+						// Default configuration-specific paths
+						// todo: move duplicated paths from conditional groups to non-conditional one
```

Comment:
That is a serious matter for different PR. I have a good idea how to implement this one though.